### PR TITLE
feat:added whishlist page

### PR DESCRIPTION
**Pull request by KanekarSnehal**
Created whish list page in which user can delete items if he no longer needs them.

### CODE REVIEW

File: pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+                </li>
+                <li class="secondary-text-color">
+                    <a href="#">Trending</a>
+                </li>
+            </ul>
+        </div>
+
+        <div class="search-input mx-auto">
+            <a class="fas fa-search search-icon"></a>
+            <label for="searchbar"></label>
```
Comment (amarphule) [inline pages/wishlist.html:38]: why it is empty?

**Review by amarphule — CHANGES_REQUESTED** [review]
looks good. don't create empty elements

File: pages/wishlist.html (hunk 1 above)
Comment (KanekarSnehal) [reply pages/wishlist.html]: Corrected it, please check it.

**Review by Prabhav-Sharma — APPROVED** [review]

**Review by Mohitkumar33 — APPROVED** [review]
right

**Pull request merged**